web/templates/settings/active_user_list_admin.hbs

```diff
@@ -0,0 +1,28 @@
+<div id="admin-active-users-list" class="user-settings-section" data-user-settings-section="user-list-admin">
```

Comment:
I think we should also rename `data-user-settings-section` here to make it consistent with what we have for deactivated users template.

Reply:
Sure. 
Moreover I think we need to rename some endpoints as well.
Because for Deactivated Users , Invitations section initially were ```"/#organization/deactivated-users-admin"``` and ```"/#organization/invites-list-admin"```. But now we show all three sections using ```"/#organization/user-list-admin"```.(For reference - ```help_settings_links.py```).
Also we might need some documentation edits.

Reply:
Yes. I forgot about those. Maybe we can have different url for each tab like we do for right section in stream settings but am not sure. Would be good to discuss it on CZO.

Reply:
Currently, we show ```"/#organization/user-list-admin"``` for all the three tabs . But if we define the tab-url based on their keys then the active users section which opens by default would have ```\user-list-admin``` initially which will then be changed to ```\active-users-admin``` on tabbing. How do we need to sort this out ?
I have tried to changed the url on tabbing but the initial url still seems an issue.